pkg/common/cns-lib/vsphere/virtualcentermanager.go

```diff
@@ -145,3 +150,34 @@ func (m *defaultVirtualCenterManager) UnregisterAllVirtualCenters(ctx context.Co
 	})
 	return err
 }
+
+func (m *defaultVirtualCenterManager) IsvSANFileServicesSupported(ctx context.Context, host string) (bool, error) {
+	log := logger.GetLogger(ctx)
+	is67u3Release, err := m.isVsan67u3Release(ctx, host)
+	if err != nil {
+		log.Errorf("Failed to identify the vCenter release with error: %+v", err)
+	}
+	return !is67u3Release, nil
+}
+
+func (m *defaultVirtualCenterManager) IsExtendVolumeSupported(ctx context.Context, host string) (bool, error) {
+	log := logger.GetLogger(ctx)
+	is67u3Release, err := m.isVsan67u3Release(ctx, host)
```

Comment:
don't we have to check esx version as well for extend volume ?

Reply:
That is another change which @shalini-b will look into. This PR main purpose is to know the status of vsan file service support. I have created these function as part of refactoring purposes only.